src/Components/Payment/SavedCreditCards.tsx

```diff
@@ -0,0 +1,181 @@
+import { BorderBox, Flex, Sans, Spinner, Theme } from "@artsy/palette"
+import { SavedCreditCardsDeleteCreditCardMutation } from "__generated__/SavedCreditCardsDeleteCreditCardMutation.graphql"
+import { CreditCardDetails } from "Apps/Order/Components/CreditCardDetails"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import React from "react"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+import { ConnectionHandler } from "relay-runtime"
+import styled from "styled-components"
+
+interface SavedCreditCardsProps {
+  creditCards: any
+  me: any
+  relay?: RelayProp
+}
+
+interface CreditCardsState {
+  isErrorModalOpen: boolean
+  isCommittingMutation: boolean
+}
+
+interface CreditCardProps {
+  creditCard?: any
+  me: any
+  relay?: RelayProp
+}
+
+export class CreditCard extends React.Component<
+  CreditCardProps,
+  CreditCardsState
+> {
+  state = { isErrorModalOpen: false, isCommittingMutation: false }
+
+  render() {
+    return (
+      <>
```

Comment:
no need to wrap

Reply:
I think it does since  it's including two components at the  root level (`<BorderBox` and `<ErrorModal`)